karbon/mocks/TestKarbonCreateClusterRegistrationWithNoCategory.yaml

```diff
@@ -0,0 +1,111 @@
+version: api.keploy.io/v1beta2
+kind: Http
+name: TestKarbonCreateClusterRegistrationWithNoCategory
+spec:
+    metadata:
+        Body: ""
+        Header: map[Accept:[application/json] Content-Type:[application/json] User-Agent:[nutanix]]
+        Proto: HTTP/1.1
+        ProtoMajor: "1"
+        ProtoMinor: "1"
+        URL: https://10.45.30.138:9440/karbon/v1-alpha.1/k8s/cluster-registrations/e33fd0fd-5673-45fa-825d-7ef869a91577
```

Comment:
consider swapping out the IPs here with a dummy value e.g. `prism-test.nutanix.com`.